2020 04 30 lp edits

widdowquinn opened the pull request.

Commit:
convert absolute to relative paths, comment out bibliography

These changes were necessary to be able to build from the GitHub
repo on an arbitrary machine.

In passing, I note that the bibliography in the headers of main.Rmd
has different identifiers for the same papers as are in the
ezclermont_refs.bib file.

Commit:
update abstract text

I tried to make this self-contained as a story, with an inbuilt justification
writing the tool, and an implicit justification for reporting it even though
there's already a tool out there.

Commit:
update Introduction text

This was already much improved on the version I'd seen. I had to
make fewer changes than I expected.

Commit:
update in silico PCR method section text

This section has a number of request in bold - they should be straightforward
restructurings, mostly putting elements into supplementary data.

Commit:
update Methods text

Commit:
update Results and Discussion text

Commit:
add note on STDIN/STDOUT to discussion

nickp60 reviewed (commented):
Makes sense -- I had converted to the abs paths so it could be build alongside the thesis, but that should definitely not be on master!

nickp60 committed:
address all but timing

Commit:
make minor change to Methods text

I think the genomes are filtered on the basis of metadata, rather
than metadata being filtered on the basis of metadata.

I'm not clear on what "sequence type" is - better to say that it's
Clermont type if that's what's intended.

Commit:
add comment on filtering method/script

Commit:
make changes for clarity to Methods text

Commit:
restore mention of Parsnp alignment for flow

Commit:
minor wording change around Biopython

widdowquinn committed:
Merge pull request #3 from nickp60/2020-04-30_LP-NW_edits

address all but timing

nickp60 commented:
Thanks so much!  I'll merge this and clean up the rest, adding the timing info
